pages/content/amp-dev/about/websites-2021.html

```diff
@@ -112,7 +112,7 @@ <h2 class="ap-h1">{{ _('Grab-and-go components') }}</h2>
         <p>
           {{ _('Want just the components without the AMP ecosystem? ')}}
           <a
-            href="{{ g.doc('/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide', locale=doc.locale).url.path }}"
+            href="{{ g.doc('../documentation/guides-and-tutorials/start/bento_guide.md', locale=doc.locale).url.path }}"
```

Comment:
im assuming relative URLs don't break anything?

Reply:
Yeah that should be fine.